packages/sanity/src/desk/components/pane/PaneLayout.tsx

```diff
@@ -50,6 +51,16 @@ export function PaneLayout(
     if (!collapsed && onExpand) onExpand()
   }, [collapsed, onCollapse, onExpand])
 
+  // Force popover position update when panes change
+  // This is needed to ensure that any open popovers is positioned correctly if a pane is toggled
+  // between collapsed and expanded. We normally use the boundary element, but that won't work
+  // inside the PTE
```

Comment:
This comment ending could read: `for popovers where the position is calculated from elements inside of the input itself` to be more clear.